akvo/templates/project_widgets2.html

```diff
@@ -29,28 +29,30 @@
                             <input type="radio" name="widget-project" value="project" checked>{% blocktrans with title=project.title %}Use the {{title}} project{% endblocktrans %}
                         </label>
                     </div>
-                    <div class="radio">
-                        <label>
-                            <input type="radio" name="widget-project" value="random">{% trans "Display a random project from an organisation" %}:
-                        </label>
-                    </div>
-                    <div class="form-group">
-                        <select class="form-control" id="widget-organisation" disabled>
-                        <option value="{{project.reporting_org.pk}}">{{project.reporting_org.name}}</option>
-                        {% for partner in project.all_partners.distinct %}
-                            {% if not partner == project.reporting_org %}
-                                <option value="{{partner.pk}}">{{partner.name}}</option>
-                            {% endif %}
-                        {% endfor %}
-                      </select>
-                    </div>
+                    {% if project.all_partners.distinct.count > 1 %}
```

Comment:
Yeah, doesn't really matter a lot I think.

The primary organisation is selected by default. In the current implementation you only see the other option when there are actually other organisations to choose from. We could also show it when there's no other organisation besides the primary organisation, although there's not a lot of added value for that :).